extra_model/_run.py

```diff
@@ -26,4 +28,4 @@ def run(input_path: Path, output_path: Path) -> None:
     if not output_path.exists():
         output_path.mkdir(parents=True)
 
-    results.to_csv(output_path / OUTPUT_FILE, encoding="utf-8", index=False)
+    results.to_csv(output_path / output_filename, encoding="utf-8", index=False)
```

Comment:
I'm not sure why it's not covered by the tests, but if that's the only problem with the PR, I'm OK with approving.

Reply:
Yeah, not sure either. I've adjusted the relevant entrypoint test accordingly [here](https://github.com/wayfair-incubator/extra-model/blob/cr105s_add_output_filename_argument/tests/test_cli.py#L64).